pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDao.java

```diff
@@ -76,15 +75,16 @@ public class KitDao {
     private static final String UPDATE_KIT_COMPLETE = "update ddp_kit set kit_complete = ? where dsm_kit_id = ?";
 
     // update various ddp_kit scan fields if they have not been set already.  If
-    // they have already been set, do nothing.
+    // kit_complete is set then the kit is considered sent, do nothing. condition on L87 avoids errors for initial scanned kits
     private static final String SET_DDP_KIT_SCAN_INFO_BY_DDP_LABEL_IF_NOT_SET_ALREADY = "UPDATE ddp_kit SET "
             + "kit_complete = 1, scan_date = ?, scan_by = ?, kit_label = ? "
             + "WHERE "
             + "dsm_kit_request_id = (SELECT dsm_kit_request_id FROM ddp_kit_request WHERE ddp_label = ?) "
             + "AND not kit_complete <=> 1 "
             + "AND deactivated_date is null "
             // sub-sub selected required in order to avoid mysql ERROR 1093 specifying target table in select
-            + "AND not exists (select * from (select 1 from ddp_kit k2 where k2.kit_label = ?) as subq)";
+            + "AND not exists (select * from (select 1 from ddp_kit k2 where k2.kit_label = ? "
+            + "AND dsm_kit_request_id != (SELECT dsm_kit_request_id FROM ddp_kit_request WHERE ddp_label = ?) ) as subq)";
```

Comment:
This is the main fix of this ticket, excluding this dsm_kit_request_id when checking if the kit label already exists